### ISSUE TITLE
sync_plans error with Django==1.11.2 - AttributeError: Can only be accessed via an instance

### ISSUE THREAD

**Issue by jay-johnson** [pinax/pinax-stripe#338]
Looks like the `sync_plans` command is having an error that sounds similar to the issues being fixed related to the new Django 1.11.2 build: https://github.com/dmkoch/django-jsonfield/issues/189#issuecomment-305778549

```
(venv) root:/opt/work/django/wsgi/server# /venv/bin/python manage.py sync_plans 
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 363, in execute_from_command_line
    utility.execute()
  File "/venv/lib/python2.7/site-packages/django/core/management/__init__.py", line 355, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/venv/lib/python2.7/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/venv/lib/python2.7/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/venv/lib/python2.7/site-packages/pinax_stripe-3.4.1-py2.7.egg/pinax/stripe/management/commands/sync_plans.py", line 11, in handle
    plans.sync_plans()
  File "/venv/lib/python2.7/site-packages/pinax_stripe-3.4.1-py2.7.egg/pinax/stripe/actions/plans.py", line 29, in sync_plans
    defaults=defaults
  File "/venv/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/venv/lib/python2.7/site-packages/django/db/models/query.py", line 459, in get_or_create
    lookup, params = self._extract_model_params(defaults, **kwargs)
  File "/venv/lib/python2.7/site-packages/django/db/models/query.py", line 521, in _extract_model_params
    property_names = self.model._meta._property_names
  File "/venv/lib/python2.7/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/venv/lib/python2.7/site-packages/django/db/models/options.py", line 887, in _property_names
    dir(self.model) if isinstance(getattr(self.model, attr), property)
  File "/venv/lib/python2.7/site-packages/django/db/models/options.py", line 887, in <setcomp>
    dir(self.model) if isinstance(getattr(self.model, attr), property)
  File "build/bdist.linux-x86_64/egg/jsonfield/subclassing.py", line 33, in __get__
AttributeError: Can only be accessed via an instance.
```

This is using the latest in the pinax-stripe repo: 

```
(venv) root:/opt/work/django/wsgi/server# pip list | grep pinax
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
pinax-stripe (3.4.1, /venv/lib/python2.7/site-packages/pinax_stripe-3.4.1-py2.7.egg)
(venv) root:/opt/work/django/wsgi/server# 
```

After rolling back to `Django 1.11.1` I was able to get the plans to sync:

```
  Successfully uninstalled Django-1.11.2
(venv) root:/opt/work/django/wsgi/server# pip install Django==1.11.1
Collecting Django==1.11.1
  Downloading Django-1.11.1-py2.py3-none-any.whl (6.9MB)
    100% |################################| 7.0MB 192kB/s 
Requirement already satisfied: pytz in /venv/lib/python2.7/site-packages/pytz-2017.2-py2.7.egg (from Django==1.11.1)
Installing collected packages: Django
Successfully installed Django-1.11.1
(venv) root:/opt/work/django/wsgi/server# /venv/bin/python manage.py sync_plans
(venv) root:/opt/work/django/wsgi/server# 
```

**Comment by om-henners** [comment]
I can report the same in Python 3.6.0. Rolling back to Django 1.11.1 also makes the difference.

**Comment by polski-g** [comment]
Bug present in 1.11.2, not in 1.11.1

**Comment by iMerica** [comment]
I'm seeing this issue in `Django==1.11.2` with `pinax-stripe==3.4.0`

**Comment by iMerica** [comment]
For anyone else blocked by this:

Django addressed this in bug [28269](https://code.djangoproject.com/ticket/28269) and fixed it in 1.11.3, but that release hasn't been cut yet. If you don't want to patch DjangoField or fork Pinax-Stripe, you can fix this issue temporarily by using 1.11.3 from their stable branch. Basically replace  `Django==1.11.2` with` git+git://github.com/django/django.git@stable/1.11.x#egg=Django` in your requirements.txt.

I'm not crazy about using a dev branch no matter how stable it might be, but it's better than downgrading IMO.

**Comment by iMerica** [comment]
Django just released 1.11.3
https://www.djangoproject.com/weblog/2017/jul/01/bugfix-release/

**Comment by paltman** [comment]
I can confirm that Django 1.11.3 resolves this issue.
